apps/vr-tests-react-components/src/stories/PresenceBadge.stories.tsx

```diff
@@ -1,16 +1,10 @@
 import { storiesOf } from '@storybook/react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 21 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27780&commitId=3d06fe314f391dac90bcaeefa8eba362676c3fd9&env=prod)

✅ There was 3 screenshots added, 0 screenshots removed, 1915 screenshots unchanged, 0 screenshots with different dimensions and 18 screenshots with visible difference.


  <details><summary><b>unknown</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|63| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|63| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|3| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|21| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|15| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|12| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|8| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|4| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|3| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|3| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|21| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|15| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|12| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|8| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|4| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|3| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.chromium.png|0| Added|
|PresenceBadge Converged - inverted background.default - High Contrast.chromium.png|0| Added|
|PresenceBadge Converged - inverted background.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>